adapters/e_volution/evolution_test.go

```diff
@@ -0,0 +1,20 @@
+package evolution
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderEVolution, config.Adapter{
+		Endpoint: "http://service.e-volution.ai/pbserver"})
```

Comment:
Nitpick: This doesn't have to be a real endpoint. We recommend using an obviously fake value so there's no pressure to keep this in sync with the read default endpoint.